MediaBrowser.XbmcMetadata/Savers/BaseNfoSaver.cs

```diff
@@ -353,7 +353,7 @@ public static void AddMediaInfo<T>(T item, XmlWriter writer)
 
                 if (!string.IsNullOrEmpty(stream.Language))
                 {
-                    // https://emby.media/community/index.php?/topic/49071-nfo-not-generated-on-actualize-or-rescan-or-identify
+                    // /community/index.php?/topic/49071-nfo-not-generated-on-actualize-or-rescan-or-identify
```

Comment:
Keep this link unchanged. It points to a forum post to provide context to the code.

Reply:
There were two of these - I'll put them back in.